DigitalLearningSolutions.Web/Helpers/DateValidator.cs

```diff
@@ -44,9 +44,9 @@ private static DateValidationResult ValidateDate(int day, int month, int year, s
             try
             {
                 var date = new DateTime(year, month, day);
-                if (date <= DateTime.Today)
+                if (date < DateTime.Today)

```

Comment:
Looks like it is indeed only used in the RegisterDelegateByCentre journey so it should fine from what I can tell. Although there is the OldDateValidator that will eventually be replaced by this that does have <= at this check rather than <.

Reply:
In that case it's ok to leave now, thanks for checking! 👍